lint/linter.py

```diff
@@ -431,9 +448,7 @@ def __init__(cls, cls_name, bases, attrs):
         if 'should_lint' in attrs:
             logger.warning(
                 "{}: Do *NOT* implement 'should_lint'. SublimeLinter will "
-                "have a breaking change here in the near future.  "
-                "Note: The 'self' you see in there is probably not the "
-                "same 'self' you see in other methods. Do *not* mutate 'self'!"
+                "have a breaking change here in the near future."
```

Comment:
Note: This is really a breaking change for plugins that implemented 'should_lint'! We had the warning *do not implement this* before, now we changed to classmethod and completely changed the call signature. So we need to err here if we see old plugin code because we removed the compatibility layer!

Reply:
Basically: it currently says 'SublimeLinter will have a *breaking* change in the near future.' The future is now after this PR.